source/reference/minio-mc/mc-version-enable.rst

```diff
@@ -0,0 +1,126 @@
+.. _minio-mc-version-enable:
+
+=====================
+``mc version enable``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc version enable
+
+
+Syntax
+------
+
+.. start-mc-version-enable-desc
+
+The :mc:`mc version enable` command enables versioning on the specified bucket.
+
+.. end-mc-version-enable-desc
+
+Mutually exclusive with :mc-cmd:`~mc version suspend` and :mc-cmd:`~mc version info`.
```

Comment:
OK so one find-replace you might have to do:

since we now have

```
.. mc:: mc version enable
```

any references to

```
:mc-cmd:`mc version enable`
```

Will fail from other builds (e.g. kubernetes) because intersphinx is going to look for that definition under `mc-cmd`, not `mc`.

It works fine in the linux build because of some weird fallback behavior I don't fully understand.

So the pass would be to do a find-replace for mc-cmd -> mc for these commands.

Reply:
fixed as many as I could find, hopefully all